src/tools/illink/src/linker/Linker.Steps/OutputStep.cs

```diff
@@ -125,7 +95,7 @@ protected virtual void WriteAssembly (AssemblyDefinition assembly, string direct
 				if (module.IsCrossgened ()) {
 					module.Attributes |= ModuleAttributes.ILOnly;
 					module.Attributes ^= ModuleAttributes.ILLibrary;
-					module.Architecture = CalculateArchitecture (module.Architecture);
+					module.Architecture = 1; // MSIL architecture which ultimately translates to AnyCPU
```

Comment:
To fix the build error:
```suggestion
					module.Architecture = (TargetArchitecture)0x0001; // MSIL architecture which ultimately translates to AnyCPU
```

Reply:
Thank you. Now I am out of office so I cannot test. Sorry and thank you for your review.